tests/dragonfly/generic_test.py

```diff
@@ -100,9 +100,12 @@ async def test_arg_from_environ(df_factory):
 
 async def test_unknown_dfly_env(df_factory, export_dfly_password):
     with EnvironCntx(DFLY_abcdef="xyz"):
-        with pytest.raises(DflyStartException):
+        try:
```

Comment:
if you think there is more pythonic way to do this I am open for suggestions @dranikpg